bin/data_processing/partition.sh

```diff
@@ -80,8 +80,8 @@ do
        mkdir -p ${output_dir}/${batch}
    fi
 
-   echo mv ${input_dir}/${files[$i]} ${output_dir}/${batch}
-   mv ${input_dir}/${files[$i]} ${output_dir}/${batch}
+   echo ln -s ${input_dir}/${files[$i]} ${output_dir}/${batch}
```

Comment:
@garlicbulbxian It's a good habit to add "" for directories/files. So, your code should be something like this:

ln -s "${input_dir}/${files[$i]}" "${output_dir}/${batch}"

But, I will close this pull request this time. I will comment again if you work on a big one with the same issue.